packages/type-utils/src/timeout.ts

```diff
@@ -0,0 +1,10 @@
+/**
+ * Hack to workaround NodeJS.Timeout vs number type issue.
+ *
+ * Some reads on the topic:
+ *  - https://stackoverflow.com/questions/45802988/typescript-use-correct-version-of-settimeout-node-vs-window
+ *  - https://guilhermesimoes.github.io/blog/making-settimeout-return-number-in-typescript
+ *
+ */
+export type TimerId = ReturnType<typeof setTimeout>;
```

Comment:
what about interval?

Reply:
indeed, we have some `ReturnType<typeof setInterval>` in our code, [e.g. here](https://github.com/trezor/trezor-suite/blob/b03e70cbc8d17e535e3d9ff5229596ee4b89e2e0/packages/blockchain-link/src/workers/electrum/client/caching.ts#L18)